edgeworkers/scripts/prod2stg.js

```diff
@@ -0,0 +1,11 @@
+const fs = require('fs');
```

Comment:
I think what we want is stg2prod?  Typically, we should be promoting changes from stage to prod only. Note: we have a way to detect isProd from the environment without hard-coding it. I'll work on that separately.

Reply:
If you update to the latest edgeworker code on stage, `isProd` is gone. You can remove the code to update that variable.